Bottom: bba9aa08e0472542b58e875c909365d9df786c11
Top:    ae99c4e7849c276f0fcc9c214862e200e24fa8a8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 14:33:38 +0000

Refresh of move-quad-edge-stuff-into

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 5f7671d..5ef4e2c 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -157,6 +157,16 @@ PCB_SRCS = \
 	hid/common/draw_helpers.h \
 	hid/common/hid_resource.c \
 	hid/common/hid_resource.h \
+	hid/common/edge3d.c \
+	hid/common/edge3d.h \
+	hid/common/face3d.c \
+	hid/common/face3d.h \
+	hid/common/object3d.c \
+	hid/common/object3d.h \
+	hid/common/quad.c \
+	hid/common/quad.h \
+	hid/common/vertex3d.c \
+	hid/common/vertex3d.h \
 	hid/hidint.h 
 
 EXTRA_pcb_SOURCES = ${DBUS_SRCS} ${GL_SRCS} toporouter.c toporouter.h
@@ -172,17 +182,7 @@ LIBGTK_GDK_SRCS= \
 LIBGTK_GL_SRCS= \
 	hid/gtk/gtkhid-gl.c \
 	hid/gtk/gui-trackball.c \
-	hid/gtk/gui-trackball.h \
-	hid/gtk/quad.c \
-	hid/gtk/quad.h \
-	hid/gtk/vertex3d.c \
-	hid/gtk/vertex3d.h \
-	hid/gtk/face3d.c \
-	hid/gtk/face3d.h \
-	hid/gtk/edge3d.c \
-	hid/gtk/edge3d.h \
-	hid/gtk/object3d.c \
-	hid/gtk/object3d.h
+	hid/gtk/gui-trackball.h
 
 GL_SRCS= \
 	hid/common/hidgl.c \
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2332d07..1dbfdac 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,8 +15,8 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
-#include "quad.h"
-#include "object3d.h"
+#include "hid/common/quad.h"
+#include "hid/common/object3d.h"
 
 #ifndef WIN32
 /* The Linux OpenGL ABI 1.0 spec requires that we define
